_posts/2021-11-24-custom-properties.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Custom Properties"
+date: 2021-11-24 12:33:53
+tags:
+- css
+- Sass
+categories:
+- css
+twitter_text: Custom Properties allow you to change the value of a variable in your css or Sass. Find out how.
+authors: "Ted Krueger"
+image:
+image_url:
+image_credit:
+---
+Have you ever wanted to change the value of a Sass variable based on screen size, a state, or for any other reason that might require changing a Sass variable? Well you can’t. Not with a Sass variable anyway, but you can by instead using a custom property. These are sometimes called css variables.
```

Comment:
"but you can by instead using" is kind of awkward - what about removing "instead"?